### PR TITLE
[#302] Modify Font Size & Color of Moment-View page

### PR DESCRIPTION
**Pull request by yasminegha**

### CODE REVIEW

File: frontend/src/styles/Caption.css [hunk 1]
```diff
@@ -1,90 +1,27 @@
...
-.caption-container-rejected {
-    width: 100%;
+.caption-container-rejected {    width: 100%;
     display: inline-block;
     margin: 3px 0;
     padding: 0.5em 0;
     background-color: rgba(196, 196, 196, 0.282);
 }
 
-.caption-highlighted {
```
Comment (tehblasian) [inline frontend/src/styles/Caption.css]: the class .caption-highlighted should not be deleted, and all of the @keyframes stuff as well.

File: frontend/src/styles/Caption.css [hunk 2]
```diff
@@ -1,90 +1,27 @@
...
+  font-size: 20px;
 }
 
-.caption-container {
-    border: solid 1px rgba(65, 65, 65, 0.8);
+.caption-content {
+    margin: auto 0;
 }
 
-.caption-container-accepted {
```
Comment (tehblasian) [inline frontend/src/styles/Caption.css]: restore this class

File: frontend/src/styles/Caption.css [hunk 3]
```diff
@@ -1,90 +1,27 @@
...
-    border: solid 1px #1DE28F;
-}
+.caption-container h1 {
+  font-family: 'Teko';
+  font-size: 18px;;
 
-.caption-container-rejected {
```
Comment (tehblasian) [inline frontend/src/styles/Caption.css]: restore this class

**Review by tehblasian — CHANGES_REQUESTED** [review]
The caption highlight animation as well as the accept/reject feedback is broken. Also, the caption font color should only be changed for the MomentViewPage, as it is unreadable on the profile page.

File: frontend/src/styles/Caption.css [hunk 4]
```diff
@@ -83,21 +85,38 @@
...
+.caption-container .caption-content h1 a{
+  font-family: 'Teko';
+  font-size: 20px;
+}
+
 .caption-content {
     margin: auto 0;
 }
 
+.caption-container h1 {
```
Comment (ZacharyBys) [inline frontend/src/styles/Caption.css]: Everywhere caption-container is defined, caption-container-accepted and caption-container-rejected should also have the same style

File: frontend/src/styles/Caption.css [hunk 5]
```diff
@@ -83,21 +85,38 @@
     border: solid 1px #f44b42;
 }
 
+.caption-container .caption-content h1 a{
```
Comment (ZacharyBys) [inline frontend/src/styles/Caption.css]: Everywhere caption-container is defined, caption-container-accepted and caption-container-rejected should also have the same style

**Review by ZacharyBys — CHANGES_REQUESTED** [review]
See Comment

File: frontend/src/styles/Caption.css [hunk 6]
```diff
@@ -83,21 +85,38 @@
...
+}
+
 .caption-content {
     margin: auto 0;
 }
 
+.caption-container h1 {
+  font-family: 'Teko';
+  font-size: 18px;;
+
```
Comment (DYu24) [inline frontend/src/styles/Caption.css]: Remove whitespace and extra semicolon

File: frontend/src/components/Header.js [hunk 7]
```diff
@@ -3,25 +3,23 @@ import React, { Component } from 'react';
...
-                break; 
-            case 4: 
-                headerStyle.fontSize = '24px'; 
-                break; 
-            default: 
-                headerStyle.fontSize = '12px'; 
+        let headerStyle = { fontSize: '10px', };
+        switch (size) {
+            case 2:
+                headerStyle.fontSize = '24px';
```
Comment (ZacharyBys) [inline frontend/src/components/Header.js]: Theres already a case where fontSize is 24px

File: frontend/src/styles/Caption.css [hunk 8]
```diff
@@ -70,22 +72,22 @@
         box-shadow: 0 0 0 0 rgb(29, 225, 143, 0);
     }
 }
-
-.caption-container {
```
Comment (ZacharyBys) [inline frontend/src/styles/Caption.css:74]: Why is this deleted? this was part of the accept/reject feedback

File: frontend/src/styles/MomentView.css [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+}
+.caption-creator-submit:active{
+    box-shadow: 1px 1.8px 1px #000000;
+    color:#1DE28F;
+}
+.caption-container .caption-content h1 a{
+  font-family: 'Teko';
+  font-size: 18px;
+}
+.caption-container h1 {
```
Comment (ZacharyBys) [inline frontend/src/styles/MomentView.css]: again, should be defined for caption-container-accepted and caption-container-rejected

File: frontend/src/styles/MomentView.css [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+  font-family: 'Teko';
+  font-size: 18px;
+}
+.caption-container {
```
Comment (ZacharyBys) [inline frontend/src/styles/MomentView.css]: Here as well

**Review by ZacharyBys — CHANGES_REQUESTED** [review]
The accept/reject feedback is broken with this change.

**Review by ZacharyBys — DISMISSED** [review]
Hm.

**Review by tehblasian — APPROVED** [review]

**Comment by tehblasian** [comment]
Looks good. I just fixed some small issues that came up after merging with dev.

**Pull request merged**